rclpy/rclpy/node.py

```diff
@@ -166,14 +167,14 @@ def __init__(
             ParameterEvent, 'parameter_events', qos_profile_parameter_events)
 
         with self.handle as capsule:
-            self._initial_parameters = _rclpy.rclpy_get_node_parameters(Parameter, capsule)
+            self._parameter_overrides = _rclpy.rclpy_get_node_parameters(Parameter, capsule)
```

Comment:
Just a suggestion: consider keeping the original name `_initial_parameters` for this variable. I think it's a bit more clear and it would better match the [description for the argument](https://github.com/ros2/rclpy/pull/363/files#diff-239a24d897ab2e601141491ac46633f9R131) if "initial parameters" retrieved by the node are overridden by "parameter overrides" given when the node is created.

Feel free to disregard this suggestion if you think it's fine like this.

Reply:
I pretty much just blindly find-replaced it and then looked around each change to update comments and docs, so none of it was super intentional. I'll have another look at this particular case.

Reply:
I kind of feel like this new name is appropriate, e.g. this snippet reads nicely to me:

https://github.com/ros2/rclpy/blob/fca845ba3113e7f43a5a1de899e13242ee1094d8/rclpy/rclpy/node.py#L364-L370

The one case where it doesn't mesh so well is here:

https://github.com/ros2/rclpy/blob/fca845ba3113e7f43a5a1de899e13242ee1094d8/rclpy/rclpy/node.py#L169-L170

But I think that's mostly due to the vaguely named `rclpy_get_node_parameters` (e.g. is that the current nodes? apparently not, so is it the parameters from the node options? no that's in rclpy only... so I guess its from YAML files, etc? It's not clear at all to me from the name).

I'm inclined to leave it, but if you or someone else still thinks it should be reverted I'm happy to do it since it's not user facing.

Reply:
Sounds good to keep it then; it wasn't a super strong opinion, and this is a good point so I'm also fine with it :+1: